src/Fixer/Import/GroupImportFixer.php

```diff
@@ -294,7 +295,7 @@ private function getNamespaceNameWithSlash(NamespaceUseAnalysis $useDeclaration)
     private function insertToGroupUseWithAlias(Tokens $tokens, int $insertIndex, NamespaceUseAnalysis $useDeclaration): int
     {
         $newTokens = [
-            new Token([\T_STRING, substr($useDeclaration->getFullName(), strripos($useDeclaration->getFullName(), '\\') + 1)]),
+            new Token([\T_STRING, (new ByteString($useDeclaration->getFullName()))->afterLast('\\')->toString()]),
```

Comment:
We could also use the [shortcut function `b`](https://github.com/symfony/string/blob/7.3/Resources/functions.php#L22-L25) everywhere:

```suggestion
            new Token([\T_STRING, b($useDeclaration->getFullName())->afterLast('\\')->toString()]),
```

Reply:
Because of this and the limited set of functions like `afterLast` needed I would prefer to not use any dependency at all. We can easily implement these functions in a singleton/zeroton utility class.